css/products.css

```diff
@@ -0,0 +1,16 @@
+@import url(/css/main.css);
+@import url(/css/main.css);
+@import url(/css/filter.css);
+@import url(/style.css);
+@import url(/css/scroll.css);
+.products-page {
+    display: grid;
+    grid-template-columns: 0.6fr 1fr 1fr 1fr;
+    grid-template-rows: 0.2fr 5fr 0.3fr;
+    grid-template-areas: "header header header header"
+        "filter main main main"
+        "filter main main main";
+
+}
```

Comment:
I think you can optimize this   
grid-template-columns: 0.6fr 1fr ;
grid-template-rows: 0.2fr 5fr 0.3fr;
grid-template-areas: "header header "
        "filter main "
        "filter main";